### PR TITLE
udpates

### PR DESCRIPTION
**Pull request by jhsiao999**
- add `code/pca.R` for running and plotting PCA
- add `code/scratch` for storing codes under development 
- did `conda install -c r r-gdata ` and `conda install -c r r-testit` for compiling `gene-filtering.Rmd`
- did git lfs to add a large file `output/gene-filtering.Rmd/eset-filtered.rdata`
- respond to an earlier comment about `image-cellcycleR.Rmd`: changed absolute path to relative path

### CODE REVIEW

File: code/scratch/slurm-41576371.out [hunk 1]
```diff
@@ -0,0 +1,68 @@
+Argument 1: 5
```
Comment (jdblischak) [inline code/scratch/slurm-41576371.out:1]: Do you really want to commit these log files? If not, I'd recommend adding `slurm-*` to `.gitignore`.

**Comment by jdblischak** [comment]
> did conda install -c r r-gdata and conda install -c r r-testit for compiling gene-filtering.Rmd

In the future, please also add these packages to `environment.yaml` so that it is easier for others to create the same setup. I'll go ahead and do this.

Also, the channel you'll want to use is `-c conda-forge`, not `-c r`. Didn't we setup your channels to handle this automatically? You should see something like this:

```
$ conda config --get channels
--add channels 'r'   # lowest priority
--add channels 'defaults'
--add channels 'conda-forge'
--add channels 'bioconda'
--add channels 'jdblischak'   # highest priority
```

> did git lfs to add a large file output/gene-filtering.Rmd/eset-filtered.rdata

This worked!

File: code/scratch/slurm-41576371.out (hunk 1 above)
Comment (jdblischak) [reply code/scratch/slurm-41576371.out:1]: I went ahead and added this.

Comment (jhsiao999) [reply code/scratch/slurm-41576371.out:1]: Yeah I don't want to commit these files. Forgot to delete them. Thanks for adding these to `.gitignore`!

**Pull request merged**